src/internal/common/oauth/request.ts

```diff
@@ -19,13 +20,15 @@ export class OAuthRequest {
 
   public readonly url: string;
   public readonly method: OAuthRequestMethods;
+  public readonly contentType?: string;
```

Comment:
@jasonma-shipstation This shouldn't be optional since the default value is `application/json` in the constructor